parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -99,6 +99,7 @@ pub struct MessageToXcm<
 
 /// Reason why a message conversion failed.
 #[derive(Copy, Clone, TypeInfo, PalletError, Encode, Decode, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(PartialEq))]
```

Comment:
We can be more specific here
```suggestion
#[cfg_attr(any(test, feature = "runtime-benchmarks"), derive(PartialEq))]
```